pkg/controller/usercleanup/user_cleanup_controller_test.go

```diff
@@ -0,0 +1,132 @@
+package usercleanup
+
+import (
+	"context"
+	"github.com/codeready-toolchain/api/pkg/apis/toolchain/v1alpha1"
+	"github.com/codeready-toolchain/host-operator/pkg/apis"
+	"github.com/codeready-toolchain/host-operator/pkg/configuration"
+	"github.com/codeready-toolchain/host-operator/pkg/metrics"
+	test2 "github.com/codeready-toolchain/host-operator/test"
+	"github.com/codeready-toolchain/toolchain-common/pkg/test"
+	"github.com/stretchr/testify/require"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+	"testing"
+	"time"
+
+	"k8s.io/client-go/kubernetes/scheme"
+)
+
+func TestUserCleanup(t *testing.T) {
+	t.Run("test that user cleanup doesn't delete an active UserSignup", func(t *testing.T) {
+		userSignup := &v1alpha1.UserSignup{
```

Comment:
Maybe I missed that but you don't set the creation timestamp, do you? Let's set the timestamp to something old enough to make sure an active signup is not deleted even if it's older than both thresholds (deactivated and unverified).

Reply:
I've set the creation timestamp to 3 years in the past.